docker-compose.yml

```diff
@@ -141,6 +141,8 @@ services:
       - ./videos:/data/videos
       - ./har:/data/har
       - ./.streamlit:/app/.streamlit
+    env_file:
+      - skyvern-frontend/.env
```

Comment:
environment variables below will override the env_file tho.

Reply:
updated